### PR TITLE
Nano: add OS support table

### PR DESCRIPTION
**Pull request by MeouSker77**
## Description

Add OS support table

### 4. How to test?
- [x] Document test: <https://yishuotestdoc.readthedocs.io/en/add-os-support/doc/Nano/Overview/support.html>
- [x] Manual test

### CODE REVIEW

File: docs/readthedocs/source/doc/Nano/Overview/support.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+| **BF16**           | ✅                    | ✅       | ⭕                  | ?       |
+| **IPEX**           | ✅                    | ✅       | ❌                  | ❌       |
+| **Multi-instance** | ✅                    | ✅       | ②                  | ?       |
+
+## Tensorflow Feature Support
+
+| Feature  | Ubuntu (20.04/22.04) | CentOS7 | MacOS | Windows |
+| -------- | -------------------- | ------- | ----- | ------- |
+| **BF16** | ✅                    | ✅       | ⭕     | ?       |
+
```
[the remark this answers is not in the document]
Comment (MeouSker77) [reply docs/readthedocs/source/doc/Nano/Overview/support.md:46]: added it

**Comment by rnwang04** [comment]
OpenVINO | OpenVINO fp32/bf16/fp16(GPU)/int8 acceleration -> OpenVINO | OpenVINO fp32/bf16/fp16/int8 acceleration on CPU/GPU/VPU ?

**Comment by MeouSker77** [comment]
> OpenVINO | OpenVINO fp32/bf16/fp16(GPU)/int8 acceleration -> OpenVINO | OpenVINO fp32/bf16/fp16/int8 acceleration on CPU/GPU/VPU ?

updated

File: docs/readthedocs/source/_toc.yml [hunk 2]
```diff
@@ -188,6 +188,8 @@ subtrees:
               title: "Troubleshooting Guide"
             - file: doc/PythonAPI/Nano/index
               title: "API Reference"
+            - file: doc/Nano/Overview/support
+              title: "OS Support"
```
Comment (Oscilloscope98) [inline docs/readthedocs/source/_toc.yml:192]: Should we put it before "API reference"?

**Review by Oscilloscope98 — APPROVED** [review]
Others LGTM

File: docs/readthedocs/source/doc/Nano/Overview/support.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+| **CUDA patch**     | ✅                    | ✅       | ✅                  | ✅       |
+| **JIT**            | ✅                    | ✅       | ✅                  | ✅       |
+| **Channel last**   | ✅                    | ✅       | ✅                  | ✅       |
+| **BF16**           | ✅                    | ✅       | ⭕                  | ?       |
+| **IPEX**           | ✅                    | ✅       | ❌                  | ❌       |
+| **Multi-instance** | ✅                    | ✅       | ②                  | ?       |
+
+## Tensorflow Feature Support
+
+| Feature            | Ubuntu (20.04/22.04) | CentOS7 | MacOS | Windows |
```
[the remark this answers is not in the document]
Comment (MeouSker77) [reply docs/readthedocs/source/doc/Nano/Overview/support.md:42]: > BTW, do we also need to mention MacOS with M-series chip somewhere to say that only basic features are supported/no acceleration on this kind of platform?

I think there is no need.

[the remark this answers is not in the document]
Comment (TheaperDeng) [reply docs/readthedocs/source/doc/Nano/Overview/support.md:42]: Yeah good point, add an additional col with MacOS (M chip) with ❌ on each row :)

[the remark this answers is not in the document]
Comment (TheaperDeng) [reply docs/readthedocs/source/doc/Nano/Overview/support.md]: Quick update: M-series chip could now (after today) run multi-process training.

**Pull request merged**